File: C++/Factorial.cpp
#include <iostream>

using namespace std;

int main()
{

    int num, fact = 1;

    cin >> num;

    for (int i = num; i > 0; i--)
    {
        if (num <= 15)
        {
            fact = fact * i;
        }
        else
        {
            cout << "Numero invalido";
        }
    }

    cout << fact;

    return 0;
}

File: Java/Factorial.java
import java.io.*;

public class Factorial {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

    public static void main(String[] args) throws Exception {

        int num = Integer.parseInt(br.readLine());
        int fact = 1;

        for (int i = num; i > 0; i--) {
            if (num <= 15) {
                fact = fact * i;
            } else {
                System.out.println("Numero invalido");
            }
        }

        System.out.println(fact);
    }
}

// https://omegaup.com/arena/problem/Calculadora-Factorial/
